app/build.gradle

```diff
@@ -130,7 +130,7 @@ dependencies {
   compile 'com.android.support:appcompat-v7:23.4.0'
   compile 'com.android.support:support-v4:23.4.0'
   compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  compile 'com.mapzen:mapzen-android-sdk:1.0.10-SNAPSHOT'
+  compile 'com.mapzen:mapzen-android-sdk:1.1.0'
```

Comment:
I'm not seeing that but I think I remember this happening before for you. I'll circle back to this issue after addressing the others.